app/assets/stylesheets/application.css.scss

```diff
@@ -75,4 +75,4 @@
 @import "moj/case_tabs";
 @import "moj/definition-list";
 @import "moj/govuk_overrides";
-@import "moj/component/govuk_warning";
```

Comment:
does moj/component/govuk_warning need to be removed?

Reply:
govuk_warning component removed.